.github/workflows/test-runtime.yml

```diff
@@ -1,8 +1,9 @@
 name: Build and test runtime
 
 on:
+  workflow_dispatch: 
   push:
-    branches: "master"
```

Comment:
why change here?

Reply:
Pushes to branches other than main should also trigger the CI, I think. This would be convenient for non-pull request debugging purpose

Reply:
It's also convenient for consuming CI time of my own account instead of the organization account. In this way I don't need a pull request to test the workflows in my own fork for branch other than main